lib/rucio/core/rule.py

```diff
@@ -1015,6 +1015,14 @@ def list_rules(
                 value = DIDType(value)
             elif key == 'grouping' and isinstance(value, str):
                 value = RuleGrouping(value)
+            elif key == 'name':
+                # First check if it is a real DID or a wildcard
```

Comment:
Could be more specific:

```suggestion
               # Check if name contains wildcard pattern and convert to SQL LIKE syntax
```